corehq/apps/commtrack/sms.py

```diff
@@ -62,6 +62,19 @@ def handle(verified_contact, text, msg=None):
     return True
 
 
+def fake_sms(user, text):
```

Comment:
might want to make it more clear this is a test util (maybe move to tests or similar)